subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/TestsFilter.kt

```diff
@@ -0,0 +1,44 @@
+package com.avito.instrumentation
+
+private val manualAnnotations = setOf(
+    "com.avito.android.test.annotations.ManualTest",
```

Comment:
you can probably ignore all the filtering besides prefixTestFilter in dynamic config

Reply:
Do you mean I can make `annotatedWith` equals to null?

Reply:
yep, i think you can, dynamic means we could possibly run everything with optional filters, specified by client

Reply:
It could break expectations from our users who ran from teamcity.
By default, we executed the only ui annotations. Is it critical?
Btw how do we filter performance tests?

Reply:
well, on the other hand you will introduce a step back in reusability of runner, is it really worth it?

Reply:
I will create task to move instrumentation-defaults to Avito
Maybe better refactor current behavior during future task?